src/ui.form-input-money/useFormatCurrency.ts

```diff
@@ -17,41 +19,47 @@ export default function useFormatCurrency(elementId, options) {
   );
 
   onMounted(() => {
-    inputElement = document.getElementById(elementId);
-    inputElement.addEventListener("input", onInput);
+    inputElement = document.getElementById(elementId) as HTMLInputElement;
 
-    onInput(formattedValue.value);
+    if (inputElement) {
+      inputElement.addEventListener("input", onInput);
+      onInput({ target: inputElement } as unknown as InputEvent);
```

Comment:
why func param has been changed? It was before: `onInput(formattedValue.value);`